openvdb/openvdb/tree/InternalNode.h

```diff
@@ -1554,6 +1631,15 @@ InternalNode<ChildT, Log2Dim>::isValueOn(const Coord& xyz) const
     return mNodes[n].getChild()->isValueOn(xyz);
 }
 
+template<typename ChildT, Index Log2Dim>
+inline bool
+InternalNode<ChildT, Log2Dim>::isValueOff(const Coord& xyz) const
+{
+    const Index n = this->coordToOffset(xyz);
+    if (this->isChildMaskOff(n)) return this->isValueMaskOn(n);
```

Comment:
why not shorten these two lines to:

`return this->isChildMaskOff(n) ? this->isValueMaskOn(n) : mNodes[n].getChild()->isValueOff(xyz);`